Include/internal/pycore_moduleobject.h

```diff
@@ -16,6 +16,12 @@ extern int _PyModule_IsPossiblyShadowing(PyObject *);
 
 extern int _PyModule_IsExtension(PyObject *obj);
 
+static inline int
+_PyModule_HasSpecializedGetAttr(PyObject* op)
+{
+    return Py_TYPE(op)->tp_getattro != PyModule_Type.tp_getattro;
```

Comment:
I'm thinking whether it's better to just leave this as is and not wrap it in a function. WDYT?

Reply:
Do you think (1) it's better to make it a macro?  Or (2) just inline that code?

Reply:
I think (2)